src/Sylius/Bundle/SettingsBundle/Manager/SettingsManager.php

```diff
@@ -35,14 +38,19 @@ class SettingsManager implements SettingsManagerInterface
     protected $schemaRegistry;
```

Comment:
You can use ServiceRegistryInterface. The SchemaRegistry should be removed. See  #4231